imports/ui/case-wizard/case-wizard.jsx

```diff
@@ -84,6 +84,9 @@ class CaseWizard extends Component {
     evt.preventDefault()
     if (!this.checkFormInvalid()) {
       const { inputValues: { mandatory, optional }, newUserEmail, newUserIsOccupant } = this.state
+      if (optional.details === '') {
```

Comment:
What would be a better approach? I chose to insert the value directly, as opposed to doing it via setState. I tried to use callback and async/await since setState didn't update state immediately and action was dispatched with an empty value.

Reply:
That sounds like a good approach :+1: Can't think of a better one